src/components/HighTable/HighTable.tsx

```diff
@@ -149,7 +146,7 @@ export default function HighTable({
   const pendingSelectionRequest = useRef(0)
   const getOnSelectRowClick = useCallback(({ tableIndex, dataIndex }: {tableIndex: number, dataIndex?: number}) => {
     if (!selection) return
-    async function onSelectRowClick(event: MouseEvent) {
+    async function onSelectRowClick(event: React.MouseEvent) {
```

Comment:
MouseEvent is already imported from react

Reply:
Indeed, but I'm in the process of moving to explicitly reference React, because MouseEvent is also a global function, and I think it's good to know, when reading the code, that it comes from React. I'll make it coherent in the rest of the code

Reply:
(done in https://github.com/hyparam/hightable/pull/71)